src/select/typeahead/TypeaheadSelect.tsx

```diff
@@ -135,7 +137,8 @@ function TypeaheadSelect({
         <TypeaheadInput
           testid={`${testid}.search`}
           customClassName={"typeahead-select__input"}
-          inputContainerRef={typeaheadInputRef}
+          inputContainerRef={typeaheadInputContainerRef}
```

Comment:
It seems we don't use the ref for the container any longer. So we can remove it.